common/src/constants.rs

```diff
@@ -29,3 +29,6 @@ pub const OPENEBS_CREATED_BY_KEY: &str = "openebs.io/created-by";
 
 /// The value to mark the creation source of a pool to be msp-operator in labels
 pub const MSP_OPERATOR: &str = "msp-operator";
+
+/// The value to mark v0 version of resources to be stored in persistent store
+pub const V0: &str = "v0";
```

Comment:
I suggest we think this through a bit. Why is for example are we not using an enum? This makes it a whole lot easier dealing with it later on, right?